City-of-Helsinki/drupal-helfi-platform-config
# UHF-3109 Helfi Charts

How to test:
- Get an instance of your choice up and running
- Checkout correct branches: `composer require drupal/helfi_platform_config:dev-UHF-3109_helfi_charts && composer require drupal/hdbt:dev-UHF-3109_helfi_charts && composer require drupal/hdbt_admin:dev-UHF-3109_helfi_charts`
- Run `make drush-updb drush-cr`
- Go to edit a basic page and add a new paragraph "Chart" / "Kaavio" to either content areas
   - Fill in the title, description and Add new media file
   - There should be an URL field for the chart. Fill in this address: `https://app.powerbi.com/view?r=eyJrIjoiYjE5OTFhMmEtMWYzNC00YjY2LTllODMtMzhhZDRiNTJiMDQ5IiwidCI6IjNmZWI2YmMxLWQ3MjItNDcyNi05NjZjLTViNThiNjRkZjc1MiIsImMiOjh9`
   - Fill in the administrational title, title and transcript fields (and check that these fields are mandatory)
   - Click on save, select the correct media item and finally save the page.
- When viewing the basic page, check that the chart gets rendered. 

Also check these:
- https://github.com/City-of-Helsinki/drupal-hdbt-admin/pull/103
- https://github.com/City-of-Helsinki/drupal-hdbt/pull/243
